astroquery/vizier/core.py

```diff
@@ -512,19 +514,21 @@ def _args_to_payload(self, *args, **kwargs):
         else:
             columns = self.columns + columns
 
-        # keyword names that can mean 'all' need to be treated separately
-        alls = ['all', '*']
+        # special keywords need to be treated separately
+        # keyword names that can mean 'all'
+        alls = ['all', '**']
         if any(x in columns for x in alls):
             for x in alls:
                 if x in columns:
                     columns.remove(x)
             body['-out.all'] = 2
+        # keyword name that means default columns
+        if '*' in columns:
+            columns.remove('*')
+            columns_default = True
+        else:
+            columns_default = False
 
-        # process: columns - always request computed positions in degrees
```

Comment:
This was included because it is the default behavior of the Vizier online interface.  I don't have a problem with removing it, but there was a reason for it.